cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -1245,10 +1285,10 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     }
                 ];
                 if (xblockInfo.isChapter()) {
-                    tabs[0].editors = [ReleaseDateEditor];
+                    tabs[0].editors = [ReleaseDateEditor, OptionalContentEditor];
```

Comment:
Could we show the editor only when the completion is enabled (`ENABLE_COMPLETION_TRACKING_SWITCH.is_enabled()`)?

Reply:
I'm not sure how to implement that. can we do that in a followup ticket?